Sponsors.md

```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
 * “Thank you” mention in facebook & Twitter page post,
 * “Thank-you” message by organizers during Meetup (not by company/individual).
 
+## Equipment Sponsor
+As we are not close to the local community of Crete we try to live stream our monthly meetups and supply VoD through Youtube. FOr this reason we use equipment which mosto of the times is not suitable for our need.
+
+So we want some equipment to support even better our live stream capabilities such as two 1080p usb cameras and a deskptop micorphone.
```

Comment:
So we would like to upgrade our equipment, to improve our live streaming capabilities. The initial equipment required is:

* Logitech C922 Pro [80 EUR]
* Logitech C922 Pro [80 EUR]
* Desktop Microphone [50 EUR]

Reply:
I would add a stand as well or even two, for both cameras. A tripod for the first ...to be floor bound and a smaller one for the desktop.

Reply:
This is a good idea, but will make the entirety of the equipment more bulky.

Reply:
And if we have a camera in the desk, we will definitely need to make sure that we can have a USB cable that works all the way to the laptop that does the live stream.

Reply:
Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa